Index: app/models/mailer.rb
===================================================================
--- app/models/mailer.rb	(revision 3735)
+++ app/models/mailer.rb	(working copy)
@@ -43,7 +43,7 @@
     message_id issue
     recipients issue.recipients
     cc(issue.watcher_recipients - @recipients)
-    subject "[#{issue.project.name} - #{issue.tracker.name} ##{issue.id}] (#{issue.status.name}) #{issue.subject}"
+    subject "[#{issue.project.name} - #{issue.tracker.name} ##{issue.id}] #{issue.subject}"
     body :issue => issue,
          :issue_url => url_for(:controller => 'issues', :action => 'show', :id => issue)
     render_multipart('issue_add', body)
@@ -67,7 +67,6 @@
     # Watchers in cc
     cc(issue.watcher_recipients - @recipients)
     s = "[#{issue.project.name} - #{issue.tracker.name} ##{issue.id}] "
-    s << "(#{issue.status.name}) " if journal.new_value_for('status_id')
     s << issue.subject
     subject s
     body :issue => issue,
